lib/puppet/file_serving/fileset.rb

```diff
@@ -47,8 +47,8 @@ def initialize(path, options = {})
       initialize_from_hash(options)
     end
 
-    raise ArgumentError.new("Fileset paths must exist") unless valid?(path)
-    raise ArgumentError.new("Fileset recurse parameter must not be a number anymore, please use recurselimit") if @recurse.is_a?(Integer)
+    raise ArgumentError.new(_("Fileset paths must exist")) unless valid?(path)
+    raise ArgumentError.new(_("Fileset recurse parameter must not be a number anymore, please use recurselimit")) if @recurse.is_a?(Integer)
```

Comment:
Can you add a note that `recurse` is an app parameter that isn't translated (same for recurselimit)?